lib/types.js

```diff
@@ -10,6 +10,9 @@ const OPTIONS_ALLOWED = [
 ];
 
 const validate = (type, v, nullable, schemas, options) => {
+  if(schemas && !Array.isArray(schemas)) {
```

Comment:
we typically space out if statements with `if () {}`

Reply:
Got it, fixed!